pkg/image/qemu.go

```diff
@@ -155,7 +155,7 @@ func (o *qemuOperations) Info(url *url.URL) (*ImgInfo, error) {
 		output, err = qemuExecFunction(qemuInfoLimits, nil, "qemu-img", "info", "--output=json", url.String())
 	}
 	if err != nil {
-		return nil, errors.Wrapf(err, "Error getting info on image %s", url.String())
+		return nil, errors.Errorf("%s, %s", output, err.Error())
```

Comment:
We now lost the URL in the error but I suppose it's okay because that information is in the spec of the DataVolume.